### ISSUE TITLE
NewTask form stays open regardless of MainWindow's state

### ISSUE THREAD

**Issue by james-gould** [james-gould/Negotium#2]
Although I'm fairly certain this is directly related to the behaviour causing issue #1 I'm opening another issue in case it's not solved in parallel to #1.

Behaviour: MainWindow should `Exit()` the application, but the process remains running if `NewTask()` was used during runtime.

**Comment by james-gould** [comment]
Fixed with https://github.com/james-gould/Negotium/commit/113d01ad23161c3fd55e0688be47f2661dafda33
